.gitignore

```diff
@@ -28,3 +28,6 @@ tsconfig.tsbuildinfo
 
 # Sentry
 .sentryclirc
+
+# IDE
+.idea
```

Comment:
```suggestion

```

Reply:
I think you don't want me to commit my 9 files in .idea folders so that I would remove it :)

just to mention - a lot of scaffold projects include this ignore by default

Reply:
Ahh gotcha, then let's go with yours 🚀